### PR TITLE
Add user-centric language to UG

### PR DESCRIPTION
**Pull request by JunCheng98**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: User Guide
 ---
-This guide serves to provide a reference for first-time users to get familiar with the `ZooKeep` app.
+If this is your first-time using `ZooKeep`, this guide will serve to provide a reference for you to get familiar with the application.
```
Comment (m0nggh) [inline docs/UserGuide.md]: I think first time should not be hyphenated?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -80,31 +81,34 @@ Format: `help`
...
+After you finish using `ZooKeep`, you can use this command to close it.
 
 Format: `exit`
 
 ---
 
 ### Listing all animals: `list`
 
-Lists all animals under the care of the user.
+This command is convenient in cases where you need to quickly reference all the animals stored in `ZooKeep` or if you
```
Comment (m0nggh) [inline docs/UserGuide.md]: Perhaps you can say 'stored in your ZooKeep book' instead to standardise with what you wrote below.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -194,7 +207,10 @@ Examples:
 
 ### Finding animals by their fields: `find`
 
-Finds and lists all animals in the `ZooKeep` book whose fields contain any of the specified argument keywords (at least 1).
+You may find yourself in situations where you need to retrieve a specific group of animals from the `ZooKeep` book. For
+example, you may need to find animals that have the same feed times or animals that have similar medical conditions.
```
Comment (m0nggh) [inline docs/UserGuide.md]: Just a suggestion: find animals with same feed times or similar medical conditions

**Review by m0nggh — COMMENTED** [review]
I think you can try to standardise to write 'your ZooKeep book' or 'the ZooKeep book' since I see variations of it in the different feature descriptions. Other than a few nitpicks, LGTM!

File: docs/UserGuide.md (hunk 3 above)
Comment (JunCheng98) [reply docs/UserGuide.md]: Fixed this and all other issues above

**Pull request merged**